packages/react-components/src/components/styles/VideoTile.styles.ts

```diff
@@ -1,12 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { IStyle, mergeStyles } from '@fluentui/react';
+import { DefaultPalette as palette, IStyle, mergeStyles } from '@fluentui/react';
 
 export const rootStyles: IStyle = {
   position: 'relative',
   height: '100%',
-  width: '100%'
+  width: '100%',
+  background: palette.neutralLighter,
```

Comment:
Extract the colors from the provided theme (via `useTheme()`) so that we respect the surrounding `FluentThemeProvider`.